tests/classes/apirest/apirest_test.php

```diff
@@ -219,7 +219,7 @@ public function test_get_credentials() {
 
         // Expect to return resdata.
         $api = new apirest($mockclient3);
-        $result = $api->get_credentials(1000, "person2@example.com");
+        $result = $api->get_credentials(1000, "PErson2@example.com");
```

Comment:
I think this upper case test can be omitted since we've already tested the functionality with `$mockclient1` and the purpose of `$mockclient3` is to test the 401 response.